SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/NewProximityIntersection.h

```diff
@@ -52,20 +45,12 @@ class SOFA_BASE_COLLISION_API NewProximityIntersection : public BaseProximityInt
 
 };
 
-} // namespace collision
+} // namespace sofa::core::collision
```

Comment:
sofa::component::collision no sofa::core

yes... I'm really reading!

Reply:
right.
(And i dont doubt about it, with your remarks on end of line or spaces 😙)